src/site/layouts/media_list_images.drupal.liquid

```diff
@@ -4,109 +4,92 @@
+                        {% endif %}
+                        <!-- Images List -->
+  
+                        <dl class="usa-unstyled-list">
+                            {% include "src/site/includes/images_list_with_additional_info_component.drupal.liquid" 
+                            with images = fieldMediaListImages.entity.fieldImages image_title_classes = image_title_classes image_title_tag = image_title_tag%}
+                        </dl>
+                        <!-- Repeated buttons -->
+                        {% if fieldButtonsRepeat %}
+                            <div class="vads-u-margin-top--3">
+                                {% for fieldButton in fieldButtons %}
+                                    {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
+                                {% endfor %}
+                            </div>
+                        {% endif %}
+                    </article>
 
-            <!-- Repeated buttons -->
-            {% if fieldButtonsRepeat %}
-              <div class="vads-u-margin-top--3">
-                {% for fieldButton in fieldButtons %}
-                  {% include "src/site/paragraphs/button.drupal.liquid" with entity = fieldButton.entity %}
-                {% endfor %}
-              </div>
-            {% endif %}
-          </article>
-
-          <!-- Tags -->
-          {% include "src/site/includes/tags.drupal.liquid" with fieldTags = fieldTags %}
+                    <!-- Tags -->
+                    {% include "src/site/includes/tags.drupal.liquid" with fieldTags = fieldTags %}
 
-          <!-- How do you rate -->
-          {% include "src/site/includes/how-do-you-rate.drupal.liquid" %}
+                    <!-- How do you rate -->
+                    {% include "src/site/includes/how-do-you-rate.drupal.liquid" %}
 
-          <!-- Related information -->
-          {% include "src/site/includes/related-information.drupal.liquid" with fieldRelatedInformation = fieldRelatedInformation %}
+                    <!-- Related information -->
+                    {% include "src/site/includes/related-information.drupal.liquid" with fieldRelatedInformation = fieldRelatedInformation %}
 
-          <!-- VA benefits -->
-          {% include "src/site/includes/benefit-hubs-links.drupal.liquid" with fieldRelatedBenefitHubs = fieldRelatedBenefitHubs %}
+                    <!-- VA benefits -->
+                    {% include "src/site/includes/benefit-hubs-links.drupal.liquid" with fieldRelatedBenefitHubs = fieldRelatedBenefitHubs %}
 
-          <!-- Need more help -->
-          {% include "src/site/paragraphs/contact_information.drupal.liquid" with entity = fieldContactInformation.entity %}
+                    <!-- Need more help -->
+                    {% include "src/site/paragraphs/contact_information.drupal.liquid" with entity = fieldContactInformation.entity %}
```

Comment:
Just so you know, I believe all `include`d templates have access to parent templates' variables by default 🙂

Reply:
That's correct @kelsonic.

Reply:
oh very cool, that's good to know. i'll remove the "prop-drilling" and see if i can get that working.